test/unit/specs/fields/fieldNoUiSlider.spec.js

```diff
@@ -36,7 +36,71 @@ describe("fieldNoUiSlider.vue", () => {
 			expect(field.$el).to.be.exist;
 
 			expect(input).to.be.defined;
-			expect(input.classList.contains("slider")).to.be.true;			
+			expect(input.classList.contains("slider")).to.be.true;
+			expect(input.disabled).to.be.undefined;
+		});
+
+		before( () => {
+			vm.$appendTo(document.body);
+		});
+
+		it("should contain an handle element", (done) => {
+			if (window.noUiSlider) {
+				vm.$nextTick( () => {
+					let handle = input.querySelector(".noUi-handle");
+					expect(handle).to.be.defined;
+					// expect(input.classList.contains("noui-target")).to.be.true;
+					done();
+				});
+			} else {
+				// eslint-disable-next-line
+				throw new Exception("Library is not loaded");
+			}
+		});
+
+		it.skip("should contain the value", (done) => {
+			vm.$nextTick( () => {
+				let origin = input.querySelector(".noUi-origin");				
+				expect(origin.style.left).to.be.within("70%", "90%");
+				done();
+			});
+		});
+
+		before( () => {
+			vm.model = { rating: 10 };
+		});
+
+		it("handle value should be the model value after changed", (done) => {
+			vm.$nextTick( () => {
+				let origin = input.querySelector(".noUi-origin");				
+				expect(origin.style.left).to.be.equal("100%");				
+				done();
+			});
+		});
+
+		// before( (done) => {
+		// 	input.querySelectorAll(".noUi-origin")[0].style.left = "0%";
+		// 	vm.$nextTick( () => {
+		// 		done();
+		// 	});
+		// });
+
+		it.skip("model value should be the handle value after changed", (done) => {
+			vm.$nextTick( () => {
```

Comment:
I'm following mocha documentation. It is better to skip a test than comment it.

> Best practice: Use .skip() instead of commenting tests out.
 
As for why exactly I skipping them, there is a problem with the value.
Without `vm.model = { rating: 10 };`, the slider don't move (`.noUi-origin` css `left` value is 0%), the value is not set at all (part of the problem come form my mistake in the schema definition, but even when fixed, there is still a problem). When placed in a `before` hook, the value is set properly. When used in a `it` statement, I need to use a `setTimeout` inside the `nextTick()` to get the changes.